src/templates/en/2019/index.html

```diff
@@ -89,6 +89,35 @@
   <link rel="stylesheet" href="/static/css/index.css">
 {% endblock %}
 
+{% block scripts %}
+  <script defer="defer" nonce="{{ csp_nonce() }}">
+    var url = '/static/images/intro-background-fit.png';
+    var setImage = function(blob) {
+      var reader = new FileReader;
+      reader.onload = function() {
+        var body = document.getElementsByTagName('body')[0];
+        body.setAttribute('style', 'background-image: url(' + reader.result + ');');
+        body.removeAttribute('class');
+      };
+
+      reader.readAsDataURL(blob);
+    };
+
+    if (window.fetch) {
```

Comment:
We could also just not do the `fetch` part, because XHR works everywhere. It just made me sad to have to write it. :P

Reply:
We use `fetch` without fallbacks in other places. Not a great idea, but there's precedent if you wanted to ignore IE.

Reply:
I don't know, it's up to you. It's a non-critical part of the application, so IE users would just see a plain grey background. Nothing would 'break' per-se.

Reply:
Let's remove it :)

![image](https://user-images.githubusercontent.com/1120896/68814297-3fafc400-062d-11ea-88ed-ec7db49bd36c.png)

IE is .3% of traffic and deprecated.

Reply:
![image](https://user-images.githubusercontent.com/105242/68816072-2a896400-0632-11ea-8dca-e2d15d05cdae.png)